howso/editing.amlg

```diff
@@ -494,6 +497,9 @@
 		(accum_to_entities (assoc
 			!dataMassChangeSinceLastAnalyze (* (size entities) (/ 1 (size !trainedFeatures)) )
 		))
+		(accum_to_entities (assoc
```

Comment:
same, should go into the accum_to_entities above to make it all one accumulation